src/Model/Post/Media.php

```diff
@@ -616,6 +621,63 @@ public static function getType(string $mimeType): int
 		return $type;
 	}
 
+	/**
+	 * Fetch video dimensions using getID3
+	 *
+	 * @param array $media
+	 * @return array media with added dimensions
+	 */
+	private static function getVideoDimensions(array $media): array
+	{
+		if (isset($media['width']) && isset($media['height']) && $media['width'] > 0 && $media['height'] > 0) {
```

Comment:
If we are not sure about the type of height/width we should explicit type cast to integer first.

```suggestion
		if (isset($media['width']) && isset($media['height']) && intval($media['width']) > 0 && intval($media['height']) > 0) {
```